orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/RunTaskHandler.kt

```diff
@@ -203,6 +211,10 @@ class RunTaskHandler(
       ).forEach { name, tags ->
         registry.timer(name, tags).record(elapsedMillis, TimeUnit.MILLISECONDS)
       }
+
+      if (elapsedMillis >= warningInvocationTimeMs) {
+        log.info("Task invocation for task ${taskModel.implementingClass} in stage ${stage.type} with id ${stage.id} took over $warningInvocationTimeMs")
```

Comment:
Might be a little easier to parse this log statement (in sumo) if:

```
log.info("Task invocation took over {}ms (taskType: {}, stageType: {}, stageId: {})", ...)
```

Short of actually dropping them on MDC.